### DISCUSSION TITLE
Not able to connect bull queue with redis cloud instance?

### DISCUSSION THREAD

**Discussion by aditodkar** [OptimalBits/bull#2223] [Q&A]
I am trying to connect redis free cloud instance with bull queue but getting error as it is not able to connect.

1) I tried below code:
```
const Bull = require("bull");
const emailQueue = new Bull("email", {
  redis: "",
});

```
For above code it is giving error Error: connect ECONNREFUSED 127.0.0.1:6379 message.

2) Also tried something like this: using tls field but did not work.
```
const Bull = require("bull");
const emailQueue = new Bull("email", {
   redis: {
     port: "",
     host: "",
     tls: { rejectUnauthorized: false },
   },
});
```
Note: I am using redis free cloud instance with bull queue and also download redis insight desktop application. I have added database to redis insight desktop app and it is connected but in node application it is not working. Am I missing any config?

**Comment by manast** [comment]
Have you tried to connect to said redis instance using ioredis directly? https://github.com/luin/ioredis

**Reply by aditodkar** [reply]
But ioredis or redis package will not be required right?

**Reply by aditodkar** [reply]
I was referring this https://stackoverflow.com/a/67612006/6213230
